src/main/java/Duke.java

```diff
@@ -1,10 +1,114 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final int DEADLINE_LENGTH = 8;
+    public static final int EVENT_LENGTH = 5;
+
+    private static Task[] tasks = new Task[100];
+    private static int numOfTask = 0;
+
+
     public static void main(String[] args) {
+        printGreetingMessage();
+        boolean isExited = false;
```

Comment:
isExited is grammatically incorrect
Suggestion:maybe you could use isExit as this will sound more like a question and it fits the English grammar better